monitoring/README.md

```diff
@@ -175,6 +175,34 @@ the following jobs in the Istio Prometheus instance:
 * `kubernetes-pods`
 * `kubernetes-service-endpoints`
 
+### Add Custom Dashboards
```

Comment:
This section describes two of the three ways to deploy additional dashboards. We should probably mention the `$USER_DIR/monitoring/dashboards' method here in addition to the section in the generic-base sample dashboards directory.